pmm/pmm-1-0-documentation-pdf.groovy

```diff
@@ -0,0 +1,56 @@
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'master',
+            description: 'Tag/Branch for build',
+            name: 'BRANCH_NAME')
+    }
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '14', numToKeepStr: '5'))
+    }
+    triggers {
+        pollSCM 'H/15 * * * *'
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                deleteDir()
+                git poll: false, branch: BRANCH_NAME, url: 'https://github.com/percona/pmm.git'
+            }
+        }
+        stage('Publish') {
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh '''
+                        sudo yum -y install docker
+                        sudo usermod -aG docker ec2-user
+                        sudo service docker start
+                        echo BRANCH = \${BRANCH_NAME}
+                        echo "Building in: " `pwd`
+                        cd doc
+                        sudo docker build . --tag pmm_1_0_doc_docker_image
```

Comment:
Why do you build docker image every time when you run the job? Maybe it is better to build it once and push it to docker hub (Percona Lab) .

Reply:
Tried, Cna't push to hub it seems, check this https://pmm.cd.percona.com/view/all/job/doc-test/43/console 
`denied: requested access to the resource is denied`